src/cbwd/instance_configuration.cc

```diff
@@ -107,9 +107,7 @@ bool instance_configuration::same_child(
  *
  *  @return  True if empty.
  */
-bool instance_configuration::is_empty() const throw() {
-  return _name.empty();
-}
+bool instance_configuration::is_empty() const throw() { return _name.empty(); }
```

Comment:
Could you change the throw() with noexcept please.
To be more C++11 compliant, the header should be
bool instance_configuration::is_empty() const noexcept {

And could you do the same for all method declarations with throw() in the header.